src/.nuget/packages.builds

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <!-- This property must be set to the same value as $(PackageOutputPath) for the nuspecs and nupkgs to be binplaced to the intended location. -->
     <OutputPath>$(PackageOutputPath)</OutputPath>
+    <PackageIndexFile>$(MSBuildThisFileDirectory)packageIndex.json</PackageIndexFile>
```

Comment:
@weshaggard is this all that's required (along with adding packageIndex.json, which is just above)?